src/main/java/mServer/crawler/sender/wdr/WdrDeserializerBase.java

```diff
@@ -0,0 +1,24 @@
+package mServer.crawler.sender.wdr;
+
+public abstract class WdrDeserializerBase extends HtmlDeserializerBase {
+    
+    private static final String URL_ROOT = "http://www1.wdr.de";
+    
+    protected enum UrlType {
+        None,
+        OverviewPage,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)